gsa/public/locales/gsa-de.json

```diff
@@ -187,8 +191,10 @@
   "CERT-Bund Advisories": "CERT-Bund-Advisories",
   "CERT-Bund Advisories Filter": "CERT-Bund Advisories-Filter",
   "CERT-Bund Advisories by CVSS (Total: {{count}})": "CERT-Bund Advisories nach CVSS (Gesamt: {{count}})",
+  "CERT-Bund Advisories by CVSS (Total: {{count}})nullplural": "",
```

Comment:
What's that?

Reply:
It's an artifact of the i18n-extract command. If there is the keyword "count" in any of the to-be-translated strings, it puts a nullplural in the end without a translation. Normally this will be removed, as long as we don't translate it, with yarn clean-up-translations. I'll clean it up once more to get rid of this.

Reply:
ok. strange thing. I could image that our setup is wrong for this kind of plural/singular thing atm. Nevertheless we should not care about this in this pr.

Reply:
Actually, if we just clean up properly after an i18n-extract again, this will not show up at all. I don't think this will ever be a problem. The clean-up script seems not to have been executed correctly. If it had, you would never have seen this ;)